content-src/components/Base/Base.jsx

```diff
@@ -162,6 +162,7 @@ export class BaseContent extends React.PureComponent {
 
     const outerClassName = [
       "outer-wrapper",
+      isDiscoveryStream && "discovery-stream-outer-wrapper",
```

Comment:
I had to add this because otherwise the search bar starts in the middle of the page then jumps to the top. Dunno what's worse.